Formula/cgit.rb

```diff
@@ -0,0 +1,23 @@
+class Cgit < Formula
+  desc "Hyperfast web frontend for Git repositories written in C"
+  homepage "https://git.zx2c4.com/cgit/"
+  url "https://git.zx2c4.com/cgit/snapshot/cgit-1.1.tar.xz"
+  sha256 "0889af29be15fc981481caa09579f982b9740fe9fd2860ab87dff286f4635890"
+  depends_on "gettext"
+  depends_on "openssl"
+
+  def install
+    system "make", "get-git"
```

Comment:
should use a resource block instead of letting build run curl

Reply:
Shall I hard-code the Git URL, or parse the Makefile for it?

Reply:
Wait, the resource is unpacked in another directory. The Makefile assumes that Git is unpacked in `./git/`. How do I symlink it to the resource directory? I don’t see anything like `Resource#path`.